h/templates/client/viewer.html

```diff
@@ -3,6 +3,7 @@
 (See gh2642 for rationale for 'ng-show="true"')
  -->
 <selection-tabs ng-if="selectionTabsFlagEnabled && !search.query() && selectedAnnotationCount() <= 0"
```

Comment:
Shouldn't the test just be `===` rather than `<=`, since the count will never be negative unless something is broken elsewhere?